Risoxas/Trevor
# Progress 99%

TODO: Fix axios request

## Description
_A thorough description what the PR does._

Jira Link: https://trevorproject.atlassian.net/browse/$JIRA_ID

## Testing Steps (run|navigate|verify)
1. Run: `npm run test`
2. Verify: All tests pass.
3. Run: `npm run start`
4. Navigate: http://localhost:3000/test-page
5. Verify: The header text says "This is the new header"

## Screenshots
_Include relevant before/after screenshots that demonstrate the change._

## Verify That You Have:
- [x] Written a good commit message.
- [x] Written adequate unit tests.
- [x] Added necessary manual testing steps above.
- [x] Added necessary screenshots.

## Before Merging
1. All status checks pass.
2. You have the necessary number of approved reviews.
3. Copy and paste the above Description into the merge Description.
4. Be sure to `Squash and Merge`.
